src/blocks/InfoCard.tsx

```diff
@@ -7,7 +7,7 @@ export interface InfoCardProps {
   subtitle?: string
   externalHref?: string
   className?: string
-  children: React.ReactNode
+  children?: React.ReactNode
```

Comment:
Not super big, but if you are up for it, I think a story that has no children could be useful